doc/util/chpl2rst.py

```diff
@@ -71,11 +71,16 @@ def gen_link(link, chapelfile):
     abspath = os.path.realpath(chapelfile)
     filename = os.path.split(chapelfile)[1]
 
-    chpl_home = os.path.realpath(os.getenv('CHPL_HOME'))
+    chpl_home = os.getenv('CHPL_MAKE_HOME')
```

Comment:
Is the idea that we expect this to always be set in the makefile that implements `make docs`?

Reply:
If you wanted to, you could write this more concisely as:

```python
chpl_home = os.getenv('CHPL_MAKE_HOME', default=os.getenv('CHPL_HOME'))
```

Reply:
> Is the idea that we expect this to always be set in the makefile that implements make docs?

Yes

> If you wanted to, you could write this more concisely as:

What we have here seems OK to me but I can do that if you want me to.

Reply:
> What we have already in the PR seems OK to me but I can do that if you want me to.

No need.